js_modules/dagster-ui/packages/ui-core/src/assets/useAssetDefinitionFilterState.oss.tsx

```diff
@@ -224,3 +224,22 @@ export function filterAssetDefinition(
 
   return true;
 }
+
+export function getAssetFilterStateQueryString() {
+  const values: Record<keyof AssetFilterType, '1'> = {
+    groups: '1',
+    kinds: '1',
+    changedInBranch: '1',
+    owners: '1',
+    tags: '1',
+    codeLocations: '1',
+    selectAllFilters: '1',
+  };
+  const params = new URLSearchParams(location.search);
```

Comment:
Might be nice to pass either the `location.search` or the constructed `URLSearchParams` as an argument, to avoid having this utility need to dig into `location`.

Reply:
we can do that if a use cases arises, would be a minor change to do then.